cogs/mod.py

```diff
@@ -1,540 +1,628 @@
+    async def _blacklist_add(self, user : discord.Member):
+        """Adds user to bot's blacklist"""
+        if user.id not in self.blacklist_list:
+            self.blacklist_list.append(user.id)
+            fileIO("data/mod/blacklist.json", "save", self.blacklist_list)
+            await self.bot.say("User has been added to blacklist.")
+        else:
+            await self.bot.say("User is already blacklisted.")
+
+    @blacklist.command(name="remove")
+    async def _blacklist_remove(self, user : discord.Member):
+        """Removes user to bot's blacklist"""
+        if user.id in self.blacklist_list:
+            self.blacklist_list.remove(user.id)
+            fileIO("data/mod/blacklist.json", "save", self.blacklist_list)
+            await self.bot.say("User has been removed from blacklist.")
+        else:
+            await self.bot.say("User is not in blacklist.")
+
+    @blacklist.command(name="show", pass_context=True)
+    async def _blacklist_show(self, ctx):
+        """Shows current blacklisted users"""
+        blckUsers = ''
+        unknown = []
+        blckList = fileIO("data/mod/blacklist.json", "load")
+        if blckList == []:
+            await self.bot.say("There are no blacklisted people.")
+        else:
+            for i in blckList:
+                member = str(discord.utils.get(self.bot.get_all_members(), id=str(i)))
+                if member is not None:
+                    blckUsers += member + ', '
+                elif member is None:
+                    unknown.append(i)
+            blckUsers = blckUsers[:-2] 
+            await self.bot.say("The following users are blacklisted: " + blckUsers)
+        if unknown != []: 
+            await self.bot.say("There are users on the blacklist that could not be found on the servers. Would you like to delete them?")
+            response = await self.bot.wait_for_message(author=ctx.message.author)
+            response = response.content.lower().strip()
+            if response == "yes" or response == "y":
+                for i in unknown:
+                    self.blacklist_list.remove(i)
+                fileIO("data/mod/blacklist.json", "save", self.blacklist_list)
+                await self.bot.say("Users have been removed from blacklist.")
+            else:
+                await self.bot.say("Alright, I'll leave them be.")
+
+    @blacklist.command(name="clear", pass_context=True)
+    async def _blacklist_clear(self, ctx):
+        """Resets the blacklist to a blank state"""
+        blklist = fileIO("data/mod/blacklist.json", "load")
+        await self.bot.say("Are you sure you want to clear the blacklist? Everyone who was previously blacklisted be able to use me that way. (yes/no)")
+        response = await self.bot.wait_for_message(author=ctx.message.author)
+        response = response.content.lower().strip()
+        if response == "yes" or response == "y":
+            for i in blklist:
+                self.blacklist_list.remove(i)

```

Comment:
could just do `self.blacklist_list = {}` instead of removing each one individually